tests/examples/wibox/awidget/prompt/vilike.lua

```diff
@@ -0,0 +1,55 @@
+local parent    = ... --DOC_NO_USAGE --DOC_HIDE
+local wibox     = require( "wibox"     ) --DOC_HIDE
+local awful     = { prompt = require("awful.prompt"),--DOC_HIDE
+                    util   = require("awful.util")}--DOC_HIDE
+local beautiful = require( "beautiful" ) --DOC_HIDE
+local terminal  = "xterm"  --DOC_HIDE
+
+    local atextbox = wibox.widget.textbox()
+
+    -- Store a list of verbs characters in a hash
+    local verbs = {
+        -- Spawn in a terminal
+        t = function(adjs, count, cmd) return {terminal, "-e", cmd} end,  --luacheck: no unused args
```

Comment:
At one place, `terminal` was replaced with `urxvt`. This is not a solution, but could be done here too (or just ignore the problem and leave it as-is).